docs/udev-configuration.md

```diff
@@ -0,0 +1,148 @@
+# Using the udev Discovery Protocol in a Configuration
```

Comment:
Should this doc be pointed to from the main README as part of the supported protocols?

Reply:
The user-guide looks to me as ONVIF specific, with a link to udev (and agreed that generic udev should be preferred, perhaps with a reference to how to adjust to udevvideo, as you mentioned earlier). Should we perhaps modify the user-guide to have the common start and then have a split per protocol, where we can add more as we support more?